socketcan_bridge/CMakeLists.txt

```diff
@@ -0,0 +1,71 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(socketcan_bridge)
+
+find_package(catkin REQUIRED COMPONENTS can_msgs roscpp socketcan_interface)
+
+catkin_package(
+  INCLUDE_DIRS include
+  LIBRARIES socketcan_to_topic topic_to_socketcan socketcan_interface roscpp can_msgs
+  CATKIN_DEPENDS can_msgs roscpp socketcan_interface
+  DEPENDS Boost
```

Comment:
No need to depend on Boost here, since you do not use it directly in your code.
You could keep it, but then you should add the dependency in package.xml as well.